apps/api/src/hyp3_api/validation.py

```diff
@@ -119,7 +136,21 @@ def get_multipolygon_from_geojson(input_file):
     return MultiPolygon(polygons)
 
 
-def validate_jobs(jobs):
+# TODO https://github.com/ASFHyP3/hyp3/issues/2442 remove this function after two burst types are merged
+def convert_single_burst_jobs(jobs: list[dict]) -> list[dict]:
+    jobs = deepcopy(jobs)
+    for job in jobs:
+        if job['job_type'] == 'INSAR_ISCE_BURST':
+            job_parameters = job['job_parameters']
+            job_parameters['reference'], job_parameters['secondary'] = (
+                [granule] for granule in job_parameters.pop('granules')
+            )
```

Comment:
I think the intention of the code is more clear if written like this suggestion. Also, a doc-string or comment that gives insight as to why this function is necessary would be great for future reference, since I'm not sure when the job type merging will happen.

```suggestion
            ref, sec = tuple(job_parameters.pop('granules'))
            job_parameters['reference'], job_parameters['secondary'] = [ref], [sec]
```

Reply:
Agreed, that's more readable. Although, you can remove the cast to `tuple`, as the value unpacking will work without it.

Reply:
Also, if you follow the link to the GitHub issue included in the comment above this function, it explains the context for why it's necessary.

Reply:
I implemented your suggestion.